week2/The system of the student's score management. v1.0.cpp

```diff
@@ -0,0 +1,160 @@
+#include <stdio.h>
+
+void readscore(int n, long student_number[], int score[]);
+void print_total_and_average(int n, int score[]);
+void rank_by_grade_in_descending_order(int n, long student_number[], int score[]);
+void print_score(int n, long student_number[], int score[]);
+void rank_by_student_number_in_ascending_order(int n, long student_number[], int score[]);
+int search_student_rank_and_grade_by_student_number(int n, long student_number[], int score[]);
+void count_the_percentage_of_the_each_category(int n, long student_number[], int score[]);
+
+int main()
+{
+	long student_number[30];

```

Comment:
This is good for this program but in real worlds, we will use `char[]` to model the student ID because student ID can be very long

Reply:
Ok，I will pay attention to that in the future.  My homework requirement is to save the student ID with long,.  That is why I use it.